agenthub/codeact_agent/action_parser.py

```diff
@@ -179,4 +180,13 @@ def check_condition(self, action_str: str) -> bool:
         return True
 
     def parse(self, action_str: str) -> Action:
-        return MessageAction(content=action_str, wait_for_response=True)
+        is_autonomous = config.agent.is_autonomous
```

Comment:
QQ: Will the config value change when running?

Reply:
Not in this PR